Makefile

```diff
@@ -12,7 +12,7 @@ SCIENCE_TFIDF_SVM_MODEL:= tfidf-svm.pkl
 SCIENCE_SCIBERT_MODEL := scibert
 SCIENCE_LABEL_BINARIZER := label_binarizer.pkl
 
-PYTHON := python3.8
+PYTHON := python3.7
```

Comment:
this goes back and forth according to the environment, maybe we should revert to python3 to not have to change often and ensure the code runs for both versions

Reply:
Agree. or an env variable that the user can define? I keep changing to 3.8 as this is the one installed in my machine currently